llvm/include/llvm/Support/CodeGen.h

```diff
@@ -54,7 +54,7 @@ namespace llvm {
   /// Type for the unique integer IDs of code generation optimization levels.
   using IDType = int;
   /// Code generation optimization level.
-  enum Level : IDType {
+  enum class Level : IDType {
```

Comment:
Would you consider dropping the `namespace CodeGenOpt` and making this `enum class CodeGenOptLevel`? Then run `s/CodeGenOpt::Level/CodeGenOptLevel/` across the codebase. It is similar to `CodeGenFileType`. I would leave `parseLevel` and `getLevel` in their namespace.

Reply:
done